server/odc-service/src/main/java/com/oceanbase/odc/service/schedule/job/DataArchiveJob.java

```diff
@@ -56,4 +61,49 @@ public void execute(JobExecutionContext context) {
             log.info("Clear job is created,");
         }
     }
+
+    @Override
+    public void initTask(DlmTask taskUnit) {
+        super.initTask(taskUnit);
+        createTargetTable(taskUnit);
+    }
+
+
+    /**
+     * Create the table in the target database before migrating the data.
+     */
+    private void createTargetTable(DlmTask dlmTask) {
+
+        if (dlmTask.getSourceDs().getDialectType() != dlmTask.getTargetDs().getDialectType()) {
+            log.info("Data sources of different types do not currently support automatic creation of target tables.");
```

Comment:
for mysql --> ob mysql mode tenant, we expect the dialectType same or not?

Reply:
We don't expect it to be the same in different modes.

Reply:
We don't expect it to be the same in different modes.

Reply:
We don't expect it to be the same in different modes.